tests/ImageSharp.Tests/Drawing/Text/DrawTextOnImageTests.cs

```diff
@@ -101,6 +107,103 @@ public class DrawTextOnImageTests
+            fnDisplayText = fnDisplayText.Substring(0, Math.Min(fnDisplayText.Length, 4));
+            TPixel color = NamedColors<TPixel>.Black;
+
+            provider.VerifyOperation(
+                ImageComparer.Tolerant(imageThreshold: 0.1f, perPixelManhattanThreshold: 20),
+                img =>
+                {
+                    img.Mutate(c => c.DrawText(text, new Font(font, fontSize), null, Pens.Solid(color, 1), new PointF(x, y)));
+                },
+                $"pen_{fontName}-{fontSize}-{fnDisplayText}-({x},{y})",
+                appendPixelTypeToFileName: false,
+                appendSourceFileOrDescription: true);
+        }
+
+        [Theory]
+        [WithSolidFilledImages(200, 100, "White", PixelTypes.Rgba32, 50, 0, 0, "SixLaborsSampleAB.woff", AB)]
+        [WithSolidFilledImages(900, 100, "White", PixelTypes.Rgba32, 50, 0, 0, "OpenSans-Regular.ttf", TestText)]
+        [WithSolidFilledImages(1100, 200, "White", PixelTypes.Rgba32, 50, 150, 100, "OpenSans-Regular.ttf", TestText)]
+        public void FontShapesAreRenderedCorrectlyWithAPenPatterned<TPixel>(
+            TestImageProvider<TPixel> provider,
+            int fontSize,
+            int x,
+            int y,
+            string fontName,
+            string text)
+            where TPixel : struct, IPixel<TPixel>
+        {
+            Font font = CreateFont(fontName, fontSize);
+            string fnDisplayText = text.Replace("\n", "");
+            fnDisplayText = fnDisplayText.Substring(0, Math.Min(fnDisplayText.Length, 4));
+            TPixel color = NamedColors<TPixel>.Black;
+
+            provider.VerifyOperation(
+                ImageComparer.Tolerant(imageThreshold: 0.1f, perPixelManhattanThreshold: 20),
+                img =>
+                {
+                    img.Mutate(c => c.DrawText(text, new Font(font, fontSize), null, Pens.DashDot(color, 3), new PointF(x, y)));
+                },
+                $"pen_{fontName}-{fontSize}-{fnDisplayText}-({x},{y})",
+                appendPixelTypeToFileName: false,
+                appendSourceFileOrDescription: true);
+        }
+
+
+        [Theory]
+        [WithSolidFilledImages(200, 100, "White", PixelTypes.Rgba32, 50, "SixLaborsSampleAB.woff", AB)]
+        [WithSolidFilledImages(900, 100, "White", PixelTypes.Rgba32, 50,  "OpenSans-Regular.ttf", TestText)]
+        public void FontShapesAreRenderedCorrectlyAlongAPath<TPixel>(
+            TestImageProvider<TPixel> provider,
+            int fontSize,
+            string fontName,
+            string text)
+            where TPixel : struct, IPixel<TPixel>
+        {
+
+            Font font = CreateFont(fontName, fontSize);
+            string fnDisplayText = text.Replace("\n", "");
+            fnDisplayText = fnDisplayText.Substring(0, Math.Min(fnDisplayText.Length, 4));
+            TPixel colorOutline = NamedColors<TPixel>.Black;
+            TPixel colorFill = NamedColors<TPixel>.Gray;
+
+            provider.VerifyOperation(
```

Comment:
This is the expected output right?

![image](https://user-images.githubusercontent.com/6835152/41508193-efeca8f6-7240-11e8-8d33-228467845380.png)

Don't we need another test? More complex path (eg. circle) but simpler brush/pen. I'm happy to add it.